client/app/components/queries/QueryEditor/index.jsx

```diff
@@ -15,7 +15,7 @@ const QueryEditor = React.forwardRef(function(
   ref
 ) {
   const [container, setContainer] = useState(null);
-  const editorRef = useRef(null);
+  const [editorRef, setEditorRef] = useState(null);
```

Comment:
Why change it to state?

Reply:
It was needed to implemented cleanup: `useEffect` does not see changes when ref value changes (e.g. when `someRef.current` changes - component will not re-render).